core/domain/story_jobs_one_off.py

```diff
@@ -66,6 +66,16 @@ class StoryMigrationOneOffJob(jobs.BaseMapReduceOneOffJobManager):
     def entity_classes_to_map_over(cls):
         return [story_models.StoryModel]
 
+    @staticmethod
+    def compute_thumbnail_size_in_bytes(thumbnail_filename, item_id):
+        """thumbnail_url = (
```

Comment:
What's happening here and why is this function here? It should be in the relevant schema version conversion function and (presumably) not be commented out.

Reply:
updated